Makefile-docker

```diff
@@ -67,12 +67,11 @@ help_submake:
 
 initialize_db:
 	rm -rf ./user-media/* ./tmp/*
-	python manage.py reset_db
+	python manage.py reset_db --noinput
```

Comment:
Although it does make sense I liked that `reset_db` asked before destroying all the things. I'd do that manually somewhere if that change is still needed.